components/authz-service/server/server.go

```diff
@@ -60,18 +60,18 @@ func NewGRPCServer(ctx context.Context,
 	dataMigrationsConfig datamigration.Config,
 	eventServiceAddress string) (*grpc.Server, error) {
 
-	// Note(sr): we're buffering one bool, as NewPostgresPolicyServer writes
+	// Note(sr): we're buffering one version struct, as NewPostgresPolicyServer writes
 	// to this before we've got readers
-	v2Chan := make(chan bool, 1)
-	switcher := NewSwitch(v2Chan)
+	vChan := make(chan api_v2.Version, 1)
+	switcher := v2.NewSwitch(vChan)
```

Comment:
Did this still need to switch from a bool at the end of the day? It's no problem, just curious.

Reply:
yeah we need to know the minor version so ProjectsAuthorized can decide if it's on v2 and should override projects in the request.